spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :done_task do
+      status {2}
```

Comment:
`2`だとどのような値か見て分からないので、enumに対応するシンボルを記載するようにしましょう。
```suggestion
      status { :done }
```

Reply:
cd07edb　statusの値をenumを使用した値　（:done）に変更しました。